src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -3,33 +3,89 @@
     "netcoreapp1.0": {
       "dependencies": {
         "NuProj": "0.10.4-beta-gf7fc34e7d8",
-        "MSBuild": "0.1.0-preview-00022",
-        "Microsoft.Build.Framework": "0.1.0-preview-00022",
-        "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
-        "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
-        "Microsoft.Build.Targets": "0.1.0-preview-00022",
-        "Microsoft.Build": "0.1.0-preview-00022",
+        "MSBuild": "0.1.0-preview-00024-160610",
+        "Microsoft.Build.Framework": "0.1.0-preview-00024-160610",
+        "Microsoft.Build.Tasks.Core": "0.1.0-preview-00024-160610",
+        "Microsoft.Build.Utilities.Core": "0.1.0-preview-00024-160610",
+        "Microsoft.Build.Targets": "0.1.0-preview-00024-160610",
+        "Microsoft.Build": "0.1.0-preview-00024-160610",
         "Microsoft.Net.Compilers.NetCore": "1.3.0-beta1-20160429-01",
-        "Microsoft.Net.Compilers.Targets.NetCore": "0.1.4-dev",
+        "Microsoft.Net.Compilers.Targets.NetCore": "0.1.5-dev",
         "Microsoft.Cci": "4.0.0-rc3-24128-00",
         "Microsoft.Composition": "1.0.30",
-        "Microsoft.NETCore.App": "1.0.0-rc3-002857",
-        "Microsoft.NETCore.TestHost": "1.0.0-rc3-24123-01",
-        "NETStandard.Library": "1.6.0-rc3-24208-04",
         "System.Diagnostics.TextWriterTraceListener": "4.0.0-rc3-24128-00",
         "System.Runtime.Serialization.Primitives": "4.1.1-rc3-24128-00",
         "Newtonsoft.Json": "7.0.1",
-        "NuGet.Client": "3.5.0-beta2-1456",
-        "NuGet.Versioning": "3.5.0-beta2-1456"
+        "Microsoft.NETCore.Targets": "1.0.1-rc3-24128-00",
+
+        "System.AppContext": "4.1.0-rc3-24128-00",
```

Comment:
The reason why I had to add all of these extra dependencies is because I removed the reference to NETStandard.Library and Microsoft.NETCore.App because they both caused the runtime to be published into the tools dir, and if I didn't manually specify all of these dependencies then the ones restored will be older versions which are brought in as secondary dependencies from msbuild and roslyn.